config/locales/hyrax.fr.yml

```diff
@@ -84,20 +84,20 @@ fr:
             title: Libération
             varies:
               any: Le déposant peut choisir n'importe quelle durée d'embargo; les baux sont autorisés
-              between: "Le déposant peut choisir l'embargo à jour:"
-              description: "Autoriser le déposant à choisir les paramètres:"
+              between: 'Le déposant peut choisir l'embargo à jour:'
+              description: 'Autoriser le déposant à choisir les paramètres:'
               embargo:
                 1yr: 1 an après le dépôt
                 2yrs: 2 ans après le dépôt
                 3yrs: 3 ans après le dépôt
                 6mos: 6 mois après le dépôt
                 select: Sélectionnez la période d'embargo.
-              period: "Le déposant peut choisir la période d'embargo jusqu'à:"
+              period: 'Le déposant peut choisir la période d'embargo jusqu'à:'
```

Comment:
items with single quotes need to be wrapped in double quotes. French has single quotes as part of the words, so you need to keep double quotes around these phrases.